data.tf

```diff
@@ -0,0 +1,33 @@
+locals {
```

Comment:
Please do `terraform fmt` in source directory.

Reply:
done.